### DISCUSSION TITLE
EdmEntityObject / EdmEntityObjectCollection to JSON

### DISCUSSION THREAD

**Discussion by pluc77** [OData/AspNetCoreOData#1335] [Q&A]
I have a dynamic model and would like to convert instances of EdmEntityObject and EdmEntityObjectCollection to JSON strings equivalent to those of OData response payloads. Can anyone offer up guidance, please?

**Comment by julealgon** [comment]
What is it that you are trying to achieve @pluc77 ? Can you share a bit more context?

**Reply by pluc77** [reply]
Hello. I am observing a dynamic model and have instantiated my own instances of EdmEntityObject and/or EdmEntityObjectCollection. Below is some psuedo code.  I'm looking to serialize / write to JSON.  Any assistance would be greatly appreciated.

var entityObject = new EdmEntityObject(MY ENTITY TYPE HERE);

//for each property, try to set the value. Ditto for expansion into navigation properties.
//entityObject.TrySetPropertyValue(name, value);

//TODO: Serialize entityObject to JSON string formatted as an OData response.
